GOVERNANCE.md

```diff
@@ -0,0 +1,69 @@
+# Governance
+
+## License
+
+The project will use [Apache 2.0 license](LICENSE) for code. To start, the
+project will not be under any foundation, and does not have immediate plans to
+do so. However, this may change in the future.
```

Comment:
Given that now it's an OpenSSF project, should we change this?

Reply:
Good catch. Removed!